src/test/java/com/auth0/json/mgmt/guardian/FactorTest.java

```diff
@@ -28,15 +27,7 @@ public void shouldDeserialize() throws Exception {
 
         assertThat(factor, is(notNullValue()));
         assertThat(factor.isEnabled(), is(true));
+        assertThat(factor.getName(), is("sms"));
```

Comment:
I don't think `name` and `trial_expired` can be changed by the client. If I'm right, then we should never serialize this values. (removing the setters should fix that). That's why the test `shouldIncludeReadOnlyValuesOnDeserialize` exists.